docs/ollama.md

```diff
@@ -1,24 +1,26 @@
 ---
-title: Ollama 
-excerpt: Setting up MemGPT with Ollama 
-category: 6580da9a40bb410016b8b0c3 
+title: Ollama
+excerpt: Setting up MemGPT with Ollama
+category: 6580da9a40bb410016b8b0c3
 ---
 
 > ⚠️ Make sure to use tags when downloading Ollama models!
 >
-> Don't do **`ollama run dolphin2.2-mistral`**, instead do **`ollama run dolphin2.2-mistral:7b-q6_K`**.
+> Don't do **`ollama pull dolphin2.2-mistral`**, instead do **`ollama pull dolphin2.2-mistral:7b-q6_K`**.
```

Comment:
I used `pull` instead of `run` here because the goal is to make sure we preload/download the model, but we're not trying to run it in the ollama CLI.

So I think this should still be `run`, not `pull`? Let me know if I'm misunderstanding the ollama commands

Reply:
You had 'run' and I changed it to 'pull'.

Because run download, loads and runs the model in the terminal. Blocking the session from further use.

Pull just downloads and prepares the model for use.

Common confusion because the ollama docs just say to use 'run' but I think it's a noob friendly kind of approach there.

But for this use-case you'd only want to pull. Since you aren't using the model in your terminal directly via ollama when using with memgpt.